### ISSUE TITLE
The acc of gcn-link-prediction.ipynb is zero.

### ISSUE THREAD

**Issue by shikew** [stellargraph/stellargraph#1843]
### Describe the bug
Hi, 

I ran the example code of gcn-link-prediction on Colab, but the acc of training and validation are both 0.

https://colab.research.google.com/github/stellargraph/stellargraph/blob/master/demos/link-prediction/gcn-link-prediction.ipynb

### Observed behavior

Epoch 1/50
1/1 - 1s - loss: 1.9855 - acc: 0.0000e+00 - val_loss: 0.7371 - val_acc: 0.0000e+00
Epoch 2/50
1/1 - 0s - loss: 0.9304 - acc: 0.0000e+00 - val_loss: 0.7873 - val_acc: 0.0000e+00
Epoch 3/50
1/1 - 0s - loss: 0.9704 - acc: 0.0000e+00 - val_loss: 0.6657 - val_acc: 0.0000e+00
Epoch 4/50
1/1 - 0s - loss: 0.6761 - acc: 0.0000e+00 - val_loss: 0.7027 - val_acc: 0.0000e+00
Epoch 5/50
1/1 - 0s - loss: 0.7154 - acc: 0.0000e+00 - val_loss: 0.6838 - val_acc: 0.0000e+00
Epoch 6/50
1/1 - 0s - loss: 0.6837 - acc: 0.0000e+00 - val_loss: 0.6915 - val_acc: 0.0000e+00
Epoch 7/50
1/1 - 0s - loss: 0.8051 - acc: 0.0000e+00 - val_loss: 0.6677 - val_acc: 0.0000e+00
Epoch 8/50
1/1 - 0s - loss: 0.6935 - acc: 0.0000e+00 - val_loss: 0.6435 - val_acc: 0.0000e+00
Epoch 9/50
1/1 - 0s - loss: 0.6399 - acc: 0.0000e+00 - val_loss: 0.6365 - val_acc: 0.0000e+00

**Comment by huonw** [comment]
Hi @shikew, thanks for filing an issue.

Unfortunately this is due to a bug in TensorFlow version 2.2 onwards: https://github.com/tensorflow/tensorflow/issues/41361. StellarGraph has fixed this in the unreleased `develop` branch, by changing `model.compile(..., metrics=["acc"])` to `model.compile(..., metrics=["binary_accuracy"])` (in #1770).

Hopefully this helps! I'm closing this is as a duplicate of #1766 (and #1786), but please feel free to reply if you have further questions or concerns!
